client/extensions/woocommerce/state/sites/settings/email/test/selectors.js

```diff
@@ -0,0 +1,234 @@
+	store_state: 'CO',
+	store_postal_code: '90250',
+	store_country: 'US',
+	store_phone: '12345678',
+	store_locale: 'en',
+	store_timezone: 'America/New_York',
+	store_currency_code: 'USD',
+	admin_email: 'bartosz@gmail.com',
+	campaign_from_name: 'wooooooo',
+	campaign_from_email: 'store@gmail.com',
+	campaign_subject: 'wooooooo',
+	campaign_language: 'en',
+	campaign_permission_reminder: 'You were subscribed to the newsletter from wooooooo',
+	mailchimp_list: '633be793c8',
+	newsletter_label: 'Subscribe to our newsletter',
+	mailchimp_auto_subscribe: '0',
+	mailchimp_checkbox_defaults: 'check',
+	mailchimp_checkbox_action: 'woocommerce_after_checkout_billing_form',
+	mailchimp_active_tab: 'api_key',
+};
+
+const wrongKeyResponse = {
+	mailchimp_api_key: '',
+	mailchimp_debugging: false,
+	mailchimp_account_info_id: '',
+	mailchimp_account_info_username: '',
+};
+
+const emailState = {
+	extensions: {
+		woocommerce: {
+			sites: {
+				123: {
+					settings: {
+						email: {
+							settings,
+							settingsRequest: false,
+							settingsRequestError: false,
+							syncStatus: {
+								last_updated_time: '2017-10-17T22:22:44',
+								store_syncing: false,
+								mailchimp_total_products: 32,
+								product_count: 32,
+								mailchimp_total_orders: 1,
+								order_count: 1,
+								account_name: 'ski',
+								mailchimp_list_name: 'ski',
+								store_id: 'a264ea0b6cbbb',
+							},
+							syncStatusRequest: false,
+							syncStatusRequestError: false,
+							resyncRequest: false,
+							resyncRequestError: false,
+							apiKeySubmit: false,
+							apiKeySubbmitError: false,
+							storeInfoSubmit: false,
+							storeInfoSubmitError: false,
+							listsRequest: false,
+							listsRequestError: false,
+							newsletterSettingsSubmit: false,
+							newsletterSettingsSubmitError: false,
+						},
+					},
+				},
+			},
+		},
+	},
+};
+
+const loadingSettings = Object.assign( {}, emailState, {
+	extensions: {
+		woocommerce: {
+			sites: {
+				123: { settings: { email: { settingsRequest: true } } },
+			},
+		},
+	},
+} );
+
+const submittingApiKey = Object.assign( {}, emailState, {
+	extensions: {
+		woocommerce: {
+			sites: {
+				123: { settings: { email: { apiKeySubmit: true } } },
+			},
+		},
+	},
+} );
+
+const invalidKey = Object.assign( {}, emailState, {
+	extensions: {
+		woocommerce: {
+			sites: {
+				123: { settings: { email: { settings: wrongKeyResponse } } },
+			},
+		},
+	},
+} );
+
+const mailChimpNoSync = Object.assign( {}, emailState, {
+	extensions: {
+		woocommerce: {
+			sites: {
+				123: { settings: { email: { syncStatus: { mailchimp_list_name: 'n/a' } } } },
+			},
+		},
+	},
+} );
+
+describe( 'selectors', () => {
+	describe( '#isRequestingSettings', () => {
+		test( 'should be false when woocommerce state is not available.', () => {
+			expect( isRequestingSettings( {}, 123 ) ).to.be.false;
+		} );
+
+		test( 'should be false when woocommerce state is loaded.', () => {
```

Comment:
Add some extra clarification here: `should be false when woocommerce state is loaded` -> `should be false when woocommerce state is loaded and there is no active request`